src/preamble.js

```diff
@@ -2223,6 +2223,7 @@ function integrateWasmJS(Module) {
       exports = instance.exports;
       if (exports.memory) mergeMemory(exports.memory);
       Module['asm'] = exports;
+      asm = exports;
```

Comment:
I think it's a good idea to remove the `var asm`, but yeah, let's not create a difference between the wasm backend and asm.js. We should do it in both (probably as a separate PR later I guess).